plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryDistributedQueries.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.AbstractTestDistributedQueries;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.sql.TestTable;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static io.trino.testing.MaterializedResult.resultBuilder;
+import static io.trino.testing.assertions.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class TestBigQueryDistributedQueries
+        extends AbstractTestDistributedQueries
```

Comment:
ATDQ is meant to be implemented by connectors supporting read/write, whcih is not the case here. That's you had to add so many suppressions. What's the benefit of having this class?
(if stays, document the _why_ in commit message)

Reply:
Good point. But CREATE and DROP are already here.

I can remove and add it back once CTAS and INSERT are also there.

Reply:
This was a veiled question -- ie what's the benefit of doing this?

Reply:
No benefit since it doesn't buy us anything today. It should be easier to add once CREATE/DROP/CTAS/INSERT are supported.